.github/workflows/build.yml

```diff
@@ -32,7 +34,7 @@ jobs:
           group-id: org.imagemagick
           artifact-id: imagemagick-distribution
           repository-url: https://nexus.alfresco.com/nexus/content/repositories/thirdparty/
-          version: ${{ steps.build.outputs.built-version }}-ci
+          version: ${{ steps.build.outputs.built-version }}-${{ steps.imagemagick.outputs.imagemagick_release }}-ci
```

Comment:
I think it's better to have -ci as a prefix to always have it in the same place, swap them:

```
${{ steps.build.outputs.built-version }}-ci-${{ steps.imagemagick.outputs.imagemagick_release }}
```